tests/rp2040/tests.gpr

```diff
@@ -0,0 +1,84 @@
+with "config/tests_config.gpr";
```

Comment:
This is the "old" Alire generated project, we should switch to the new one but I guess you re-used the existing `armv6m` test. Don't worry about this I will do a clean up after the merge.

Reply:
That's right - I reused the `armv6m` test to make sure they were consistent.